e2e/nfs-server/ks.cfg

```diff
@@ -0,0 +1,40 @@
+# Use text install
+text
+
+# Keyboard layouts
+keyboard --vckeymap=us --xlayouts='us'
+# System language
+lang en_US.UTF-8
+
+# Firewall configuration
+firewall --use-system-defaults
+
+# OSTree setup
+ostreesetup --osname="fedora-iot" --remote="fedora-iot" --url="file:///ostree/repo" --ref="fedora/stable/x86_64/iot" --nogpg
+
+# Run the Setup Agent on first boot
+firstboot --enable
+
+# Partitions
+ignoredisk --only-use=sda
+clearpart --all --initlabel --drives=sda --disklabel=gpt
+autopart --type=btrfs
+
+# Time
+timesource --ntp-server=clock.corp.redhat.com
+timezone Europe/Berlin --utc
+
+# Authentication
+rootpw --lock
+sshkey --username=root 'sk-ssh-ed25519@openssh.com AAAAGnNrLXNzaC1lZDI1NTE5QG9wZW5zc2guY29tAAAAIB6oDRf2no5vPEM9ERg2n9ZT9Wpug/TAny/xjKgd+madAAAABHNzaDo= lis'
```

Comment:
I left it here on purpose.  I considered auto-generating the kickstart to use the ssh-keys from the ansible directory but then I thought "I have more interesting things to work on right now"

But honestly: by the time we're doing this again, a hard-coded ssh key is likely to be the least of our problems.  Probably we're switching to coreos or so and we need to replace the entire kickstart with an ignition file.

This commit isn't meant so much as "automated installer" so much as a way to avoid losing the knowledge gained during this round...